CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- **CUMULUS-260: "PDR page on dashboard only shows zeros."** The PDR stats in LPDAAC are all 0s, even if the dashboard has been fixed to retrieve the correct fields.  The current version of pdr-status-check has a few issues.
```

Comment:
I think this is part of the 1.2.0 release: https://github.com/cumulus-nasa/cumulus/pull/256/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR21

Reply:
removed duplicate entry.